src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotesJsonConverter.java

```diff
@@ -10,6 +10,8 @@
  */
 public final class ColumnAdapterNotesJsonConverter {
     private static final ColumnAdapterNotesJsonConverter COLUMN_ADAPTER_NOTES_JSON_CONVERTER = new ColumnAdapterNotesJsonConverter();
+    static final String JDBC_DATA_TYPE = "jdbcDataType";
```

Comment:
Why is one variable public and another package-private? Do you need only one of them outside the package?

Reply:
Maybe it's better to use a getter if you need a public variable?

Reply:
The second one is used in a whitebox unit test.